salt/metalk8s/calico/installed.sls

```diff
@@ -1,5 +1,12 @@
-# TODO: Configure and use local repo
+{%- from "metalk8s/map.jinja" import repo with context %}
+
+include:
+  - metalk8s.repo
+
 Install calico-cni-plugin:
   pkg.installed:
-    - sources:
-      - calico-cni-plugin: /srv/scality/{{ saltenv }}/packages/scality-el7/x86_64/calico-cni-plugin-3.5.1-1.el7.x86_64.rpm
+    - name: calico-cni-plugin
+    - version: {{ repo.packages['calico-cni-plugin'].version }}
+    - fromrepo: {{ repo.repositories.keys() | list | join(',') }}
```

Comment:
minor: I don't think the `| list` filter is necessary here (and in every other `fromrepo` you added), `join` should handle the `dict_keys` object with ease.

Reply:
Not necessary in python2, didn't you tell me that's necessary in python3 ?

Reply:
The difference is when you try to use the `dict_keys` as a list, e.g. `my_dict.keys()[0]`, which would work in Python 2 but not 3. However, `join` works with any kind of iterable, so you should be fine here.